### PR TITLE
Add docs for Delivery API range query feature in V13

### PR DESCRIPTION
**Pull request by kjac**
## Description

Added docs for the new Delivery API range query feature in V13.

Note that the "Filter operators" section from https://github.com/umbraco/UmbracoDocs/pull/5592 has not been included in the V13 docs yet. Since this feature requires changes to said section, I have included it in this PR 😄 

## Type of suggestion

* [ ] Typo/grammar fix
* [ ] Updated outdated content
* [x] New content
* [ ] Updates related to a new version
* [ ] Other

## Product & version (if relevant)

V13.0

## Deadline (if relevant)

Ideally on December 14 when V13.0 is out 😄 but if it's a few days later that's fine too. This is not a major feature.

### CODE REVIEW

**Review by alina-tincas — CHANGES_REQUESTED** [review]
small change

File: 13/umbraco-cms/reference/content-delivery-api/extension-api-for-querying.md [hunk 1]
```diff
@@ -230,6 +230,22 @@ The principal difference from the selector is that the filter implements `BuildF
 
 Since we need to perform an exact match, the index field (`authorId`) is once again defined as a "raw" string. Other options include "analyzed" and "sortable" strings. These support "contains" searches and alpha-numeric sorting, respectively.
 
+### Filter operators
+
+When implementing a filter, you can use the following operators:  `Is`, `IsNot`, `Contains`, `DoesNotContain`, `GreaterThan`, `GreaterThanOrEqual`, `LessThan` and `LessThanOrEqual`.
+
+{% hint style="info" %}
+The range operators (the latter four) only work with number and date fields - `FieldType.Number` and `FieldType.Date` respectively.
```
Comment (elit0451) [inline 13/umbraco-cms/reference/content-delivery-api/extension-api-for-querying.md:238]:
```suggestion
The range operators (_the latter four_) only work with number and date fields - `FieldType.Number` and `FieldType.Date` respectively.
```

File: 13/umbraco-cms/reference/content-delivery-api/extension-api-for-querying.md [hunk 2]
```diff
@@ -230,6 +230,22 @@ The principal difference from the selector is that the filter implements `BuildF
...
+{% endhint %}
+
+It is possible to pass multiple values to each operator, and these values will be treated inclusively as an __or__ operator. For example, if `tag1` and `tag2` were passed into a filter using the `Is` operator, _any_ document containing __either__ `tag1` __or__ `tag2` would return. The request for this might look like this:
+
+`GET /umbraco/delivery/api/v2/content?filter=customTagFilter:tag1,tag2`
```
Comment (elit0451) [inline 13/umbraco-cms/reference/content-delivery-api/extension-api-for-querying.md:243]: Let's use the <code>```http</code> format as the other examples:

```suggestion
```http
GET /umbraco/delivery/api/v2/content?filter=customTagFilter:tag1,tag2
```
```

File: 13/umbraco-cms/reference/content-delivery-api/extension-api-for-querying.md [hunk 3]
```diff
@@ -230,6 +230,22 @@ The principal difference from the selector is that the filter implements `BuildF
...
+
+If you require this functionality to be restrictive i.e. `tag1` __and__ `tag2`, then the current approach would be to chain the custom filter. The request would change to look more like this:
+
+`GET /umbraco/delivery/api/v2/content?filter=customTagFilter:tag1&filter=customTagFilter:tag2`
```
Comment (elit0451) [inline 13/umbraco-cms/reference/content-delivery-api/extension-api-for-querying.md:247]: Same here:

```http
GET /umbraco/delivery/api/v2/content?filter=customTagFilter:tag1&filter=customTagFilter:tag2
```

**Review by elit0451 — CHANGES_REQUESTED** [review]
The content is great, I've just added a few comments

**Review by elit0451 — APPROVED** [review]
Changes look great!

**Comment by jonat123** [comment]
@kjac Will merge this PR in so it is part of the v13 documentation tomorrow when we go live with it 😄

**Pull request merged**